exec/cnex/cnex.c

```diff
@@ -70,6 +70,7 @@ BOOL ParseOptions(int argc, char* argv[])
                 exit(1);
             } else if (argv[nIndex][1] == 'D') {
                 gOptions.ExecutorDisassembly = TRUE;
+                gOptions.ExecutorDebugStats = TRUE;
```

Comment:
Why is `ExecutorDebugStats` set to true here? Isn't that already covered by the `-d` switch? What if you only want to set `ExecutorDisassembly`, there's no longer a way to do it.